### DISCUSSION TITLE
Author Dropdown has Disappeared?

### DISCUSSION THREAD

**Discussion by jpmuldoon** [WordPress/gutenberg#50298] [General]
Just updated to 15.7.0.
More fool me.
The author dropdown box has disappeared.
This software tries my patience.

**Comment by t-hamano** [comment]
Hi @jpmuldoon,

To confirm this issue, can you tell me about the following?

- Does it reproduce if you update to the latest Gutenberg plugin?
- Does it reproduce even if the Gutenberg plugin is disabled?
- Does it reproduce even if you stop all plugins and switch to the default theme?
